data/core/units/nagas/Sicarius.cfg

```diff
@@ -16,15 +16,13 @@
     {AMLA_DEFAULT}
     cost=46
     usage=fighter
-    description= _ "While they sometimes still stand in as sellswords for potential Dunefolk allies, Naga Sicarii are more often the keepers of trade routes and resource beds close to waterways. For the right fee or exchange of goods, a Sicarius will guarantee safe travel or free access to valuable supplies in his territory. Rubbed the wrong way and a Sicarius becomes a fearsome foe, not because of their personal strength in combat, but instead due to the numerous allies that he can call upon to quash any potential nuisance. Though perfectly capable warriors when the time is necessary, these experienced mercenaries know perfectly well that the best methods for generating money are those that do not place themselves in danger."
+    description= _ "While they sometimes still stand in as sellswords for potential Dunefolk allies, Naga Sicarii are more often the keepers of trade routes and resource beds close to waterways. For the right fee or exchange of goods, a Sicarius will guarantee safe travel or free access to valuable supplies in his territory. Rubbed the wrong way and a Sicarius becomes a fearsome foe, not because of their personal strength in combat, but instead due to the numerous allies that he can call upon to quash any potential nuisance. Though perfectly capable warriors when the time is necessary, these experienced mercenaries know perfectly well that the best methods for generating money are those that do not place themselves in danger.  When they do find themselves in danger, their manipulative cunning is put to good use, for they can anticipate most enemy strikes, and use their heavy, curved blade to deflect or distract at the critical moment."
```

Comment:
I think "most" sounds a bit strong; how about "many" instead?

Reply:
Sure, I was just trying to make it not be "all".  This is compatible with marksman (makes it 50%) but I'm not sure how this interacts with noncumulative (magical), need to test a bit more.